prow/plugins/lgtm/lgtm.go

```diff
@@ -74,18 +75,18 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) err
 	isAssignee := ic.Issue.IsAssignee(commentAuthor)
 	isAuthor := ic.Issue.IsAuthor(commentAuthor)
 	if isAuthor && wantLGTM {
-		resp := "you can't LGTM your own PR"
+		resp := "Cannot add LGTM to your own PR"
 		log.Infof("Commenting with \"%s\".", resp)
 		return gc.CreateComment(org, repo, number, plugins.FormatResponse(ic.Comment, resp))
 	} else if !isAuthor {
-		if !isAssignee && wantLGTM {
-			resp := "you can't LGTM a PR unless you are an assignee"
-			log.Infof("Commenting with \"%s\".", resp)
-			return gc.CreateComment(org, repo, number, plugins.FormatResponse(ic.Comment, resp))
-		} else if !isAssignee && !wantLGTM {
-			resp := "you can't remove LGTM from a PR unless you are an assignee"
-			log.Infof("Commenting with \"%s\".", resp)
-			return gc.CreateComment(org, repo, number, plugins.FormatResponse(ic.Comment, resp))
+		if !isAssignee {
+			log.Infof("Assigning %s/%s#%d to %s", org, repo, number, commentAuthor)
+			if err := gc.AssignIssue(org, repo, number, []string{commentAuthor}); err != nil {
```

Comment:
Actually that's a reasonable point. By doing things this way we are essentially saying "if you say `/lgtm` then you should be assigned to the PR." We could just add the label and if they want to be assigned then they say `/lgtm /assign`.